crates/rpc/rpc/src/trace.rs

```diff
@@ -396,7 +399,7 @@ where
         &self,
         block_id: BlockId,
     ) -> EthResult<Option<Vec<LocalizedTransactionTrace>>> {
-        let traces = self
+        let mut traces: Option<Vec<LocalizedTransactionTrace>> = self
```

Comment:
perf: can use `futures::try_join` with the cache call

Reply:
let's do in follow-up as i'd like to unblock this asap